sanket/HTML/Day6/styles.css

```diff
@@ -0,0 +1,36 @@
+.bg {
+  background: tomato;
```

Comment:
full name use kariye like background-color ya background-images ok

Reply:
i don't think any one should follow the full name coding standard because its make our code more verbose and every bit in file size matters because it takes more time to download your page so one should avoid using full name.